### DISCUSSION TITLE
Changing messaging from packwerk check and package-todo headers?

### DISCUSSION THREAD

**Discussion by timjnh** [Shopify/packwerk#393] [Q&A]
Is it possible to change the messaging from `packwerk check`?  Specifically I'm interested in updating the "help" URL to something custom.  Currently it points to the packwerk docs in GitHub but I'd like to use something more tailored to my org.

> To receive help interpreting or resolving this error message, see: https://github.com/Shopify/packwerk/blob/main/TROUBLESHOOT.md#Troubleshooting-violations

I think this may be possible with a custom checker but a simpler way would be awesome.

I'm also interested in changing the documentation in the header for package-todo.yml files which I can't see anyway to override, even via code.  Thanks!

**Comment by gmcgibbon** [comment]
It isn't currently possible without forking, but we could probably make it changeable via config. PRs welcome!

**Reply by alexevanczuk** [reply]
If I'm understanding this correctly, it's possible to do this with a custom formatter! https://github.com/Shopify/packwerk/blob/main/USAGE.md#custom-offense-formatter

This is what we did while I was at Gusto to point to our internal docs on packwerk.

**Reply by gmcgibbon** [reply]
@alexevanczuk is correct, this is actually possible via an offense formatter. My mistake.

**Reply by timjnh** [reply]
Oh interesting, I missed the offense formatter.  Will give that a try.  Thanks both!

Any ideas about changing the headre in package-todo.yml?

**Reply by alexevanczuk** [reply]
That's not possible today to my knowledge @timjnh , but don't see why it couldn't be configurable if you wanted to throw a PR together!
